docs/docs/01-quickstart.md

```diff
@@ -186,8 +190,7 @@ POST to `/v1/aspects` using the following body:
   "warningRange": [2000000001, 4000000000],
   "infoRange": [4000000001, 5500000000],
   "okRange": [5500000001, 10000000000],
-  "tags": [
-    { "name": "economic" }
+  "tags": [ "economic" ]
   ],
```

Comment:
Remove closing bracket as well, since you have already added it in line 193?

Reply:
Thanks.  Submitted a new commit to correct that.  Also, submitted salesforce-cla form.